src/powfacpy/base_interface.py

```diff
@@ -862,6 +862,18 @@ def create_comtrade_obj(self,file_path: str,parent_folder=None):
     # intcomtrade.Load() probably not required
     return intcomtrade
 
+  def _handle_possible_attribute_not_specified_error(self, 
+                              potentially_not_secified_attr,
```

Comment:
`possibly_not_specified_attribute`
would be more consistent with function name (no abbreviation, possible instead of potential)

Reply:
ok.
I also changes the method name: _handle_possible_attribute_not_set_error